src/components/auth/CompanyForm.tsx

```diff
@@ -1,56 +1,132 @@
-import {View} from 'react-native';
+import {TouchableOpacity, View} from 'react-native';
 import React from 'react';
+import * as Yup from 'yup';
+import {Formik} from 'formik';
+
 import TextComponent from '../TextComponent';
 import InputComponent from '../InputComponent';
-import useQuery from '../../hook/useQuery';
-import useAuth from '../../hook/useAuth';
 import {globalStyles} from '../../theme/globalTheme';
 
-const CompanyForm = () => {
-  const {user, setUser} = useQuery();
-
-  const {
-    email,
-    phone,
-    password,
-    setPassword,
-    setConfirmPass,
-    confirmPass,
-    handleEmailOrPhoneChange,
-  } = useAuth();
+const SigupCompanyScheme = Yup.object().shape({
+  name: Yup.string()
+    .min(4, 'Too short name')
+    .max(30, 'Too long name')
+    .required('Required'),
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string()
+    .min(8, 'Too short password')
+    .max(20, 'Too long password')
```

Comment:
Heres the same how is a password to long

Reply:
thanks.
I have made the changes